README.md

```diff
@@ -1,48 +1,170 @@
 <p align="center">
-  <a href="https://www.django-unicorn.com/"><img src="https://www.django-unicorn.com/static/img/unicorn.svg" alt="django-unicorn logo" height="80"/></a>
+  <a href="https://www.django-unicorn.com/"><img src="img/gu-logo.png" alt="django-unicorn logo" height="200"/></a>
 </p>
-<h1 align="center"><a href="https://www.django-unicorn.com/">Unicorn</a></h1>
-<p align="center">The magical full-stack framework for Django ✨</p>
+
+# Django Unicorn ✨
```

Comment:
I tend to call it just `Unicorn` because my understanding is that using "Django" in the name requires permission from the actual Django organization.

Reply:
Oh interesting. Didn't know that.